apps/flows/src/app/transaction-details-flow/for-you/for-you.component.html

```diff
@@ -0,0 +1,19 @@
+<kirby-page title="Til dig" defaultBackHref="/">
+  <kirby-page-content>
+    <div class="container">
+      <div class="app-header"></div>
+
+      <kirby-card class="spacing" [hasPadding]="true"
+        >Card 1
+        <kirby-chart type="column" [data]="ChartData" [labels]="ChartLabels"></kirby-chart>
+      </kirby-card>
+      <kirby-card class="spacing" [hasPadding]="true">Card 2</kirby-card>
+      <kirby-card class="spacing" [hasPadding]="true">Card 3</kirby-card>
+      <div>
```

Comment:
Why do we need this `<div>`?

Reply:
I wanted to add a grid layout, but i already did that in another branch. I used the style spacing for now.